laravel/resources/views/partials/warning/concurrent-slot.blade.php

```diff
@@ -0,0 +1,11 @@
+<span>
+@if(isset($admin) && $admin === true)
```

Comment:
why not just use !empty($admin)?

Reply:
its just extra safety in case $admin isn't a bool